src/goto-programs/class_identifier.h

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************\
+
+Module: Extract class identifier
+
+Author: Chris Smowton, chris.smowton@diffblue.com
+
+\*******************************************************************/
+
+#ifndef CPROVER_GOTO_PROGRAMS_CLASS_IDENTIFIER_H
+#define CPROVER_GOTO_PROGRAMS_CLASS_IDENTIFIER_H
+
+#include <util/expr.h>
+#include <util/std_types.h>
+#include <util/namespace.h>
```

Comment:
Nit picking: I suppose the debate whether or not to use forward declarations between @peterschrammel and myself never concluded, so I'm still in favour of
```
class exprt;
class namespacet;
class symbol_exprt;
```